tesseract/tesseract_planning/tesseract_motion_planners/src/trajopt/trajopt_motion_planner.cpp

```diff
@@ -114,7 +115,7 @@ tesseract_common::StatusCode TrajOptMotionPlanner::solve(PlannerResponse& respon
   if (verbose)
     util::gLogLevel = util::LevelDebug;
   else
-    util::gLogLevel = util::LevelInfo;
+    util::gLogLevel = util::LevelWarn;
```

Comment:
Should we just pass the level in? I say that because I often find LevelInfo useful but LevelDebug distracting with not very useful information. However I also realize someone might want LevelWarn if they want the best performance. There is a big gap between warn with basically nothing and debug which has a bunch of marginally useful information.

Reply:
I think adding it to the base config would be good idea.